sdcm/cluster_k8s/__init__.py

```diff
@@ -185,10 +188,55 @@ def _scylla_operator_chart_version(self):
 
     @log_run_info
     def deploy_scylla_manager(self) -> None:
+        # Calculate options values which must be set
+        #
+        # image.tag                  -> self.params.get('mgmt_docker_image').split(':')[-1]
+        # scylla.scyllaImage.tag     -> self.params.get('scylla_version')
+        # scylla.agentImage.tag      -> self.params.get('scylla_mgmt_agent_version')
```

Comment:
let's use the default Scylla version as in "standard" manager installation

Reply:
done